src/app/components/notifier/mute-notifier.service.js

```diff
@@ -27,6 +30,16 @@
       info("Trying to say something? You are muted.");
     }
 
+    function joinedMuted(){
+      info("You're muted. Unmute yourself to join the conversation!");
```

Comment:
I would like something more explicit about the reason (and we don't need the "unmute yourself" part because we already have a button saying "unmute"). Something like this (just my first attempt SURE you can improve it):
"Joining to a conversation with more than X participants. You'll start muted."